CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ---
 
+## master
+
+* Fix issue with txt files bailing on comments
```

Comment:
You should end the line with a full stop + 2 spaces so that your name and the PR link are on next line in markdown :wink:
(You can get inspiration from other entries)